deltachat/rpc_test.go

```diff
@@ -11,5 +11,5 @@ func TestRpc_Start(t *testing.T) {
 	manager := acfactory.NewAcManager()
 	defer manager.Rpc.Stop()
 
-	assert.NotNil(t, manager.Rpc.Start())
+	assert.NoError(t, manager.Rpc.Start())
```

Comment:
unrelated change, NotNil is used in similar situations in several other parts, so a subsequent PR improving this and all the other usages makes sense

btw NotNil here means "it failed" so `NoError` is wrong?

Reply:
This is actually correct as it asserts for "no error" where non-nil error is an error. It was a leftover from debugging tests.

Reply:
it asserts for error, if the RPC is already started calling Start() again should return an error, that is why it must be non-nil